lineman/app/components/start_menu/start_menu.haml

```diff
@@ -0,0 +1,9 @@
+.lmo-start-menu
```

Comment:
Nah, I'll change it. I guess I was thinking 'start-menu' seems pretty clash-able, but adding lmo to the beginning of it doesn't really help that any.